### ISSUE TITLE
Bash completion:  incorrect actions presented after --vpcs resource name

### ISSUE THREAD

**Issue by fstab50** [fstab50/ec2cli#5]
```
$ ec2cli   --vpcs <tab><tab>

attach  create  list    run   
```

Only **list** is correct action that can be taken

**Comment by fstab50** [comment]
Fixed today for release version ```2.4.11```

**Comment by fstab50** [comment]
Reopen -- Failure in current release v`2.4.12`

**Comment by fstab50** [comment]
Fixed in release **2.4.14**
